go/ai/generate.go

```diff
@@ -215,10 +250,12 @@ func GenerateWithRequest(ctx context.Context, r api.Registry, opts *GenerateActi
 	}
 
 	m := LookupModel(r, opts.Model)
+
 	if m == nil {
 		return nil, core.NewError(core.NOT_FOUND, "ai.GenerateWithRequest: model %q not found", opts.Model)
 	}
 
+	isLongRunning := SupportsLongRunning(r, opts.Model)
```

Comment:
Do we need this function? Can't we just try to look up the model and then try background model? It's a long and brittle function and we are able to get that information in other ways.